charts/openshift-metering/values.yaml

```diff
@@ -669,6 +678,34 @@ hive:
           maxRAMPercentage: null
           minRAMPercentage: null
 
+        tls:
+          # server cert
+          certificate: ""
+          # server private key
+          key: ""
+          # CA for the server cert
+          caCertificate: ""
+
+          enabled: false
+          createSecret: false
+          secretName: ""
+
+        auth:
+          enabled: false
+
+        metastoreTLS:
```

Comment:
I wonder if we can rename this field to something else now that we've deflated the `reporting-operator.spec.config.prestoTLS` into `reporting-operator.spec.config.presto.tls` (and like-wise for `presto.auth`). 

That being said, I'm not sure you need a `metastore` field itself if we're only configuring TLS for metastore.

Reply:
Yeah, it's basically that. And it doesn't make much since to put into `spec.hive.metastore` either, since this needs to be done for each client of metastore, including presto. For Presto, I put it's TLS config into `spec.presto.config.connectors.hive.tls`.